crates/orderbook/openapi.yml

```diff
@@ -443,6 +443,30 @@ paths:
                   $ref: "#/components/schemas/Order"
         400:
           description: Problem with parameters like limit being too large.
+  /api/v1/token/{token}/native_price:
+    get:
+      summary: Get native price for the given token.
```

Comment:
Can we add more to this description describing exactly what a "native price" is? Maybe something about it being the exchange rate between the specified token and the network's native currency. Also, its important that this is the price per "atom" and not per "unit" (i.e. USDC has a very high price, DAI has a low price, despite them both representing $1).

Reply:
Fixed.